scss/footer.scss

```diff
@@ -0,0 +1,24 @@
+.footer-page {
+  align-items: center;  
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+    
```

Comment:
I think is necessary a `padding` because currently is different to the design in the image of Trello